CMakeLists.txt

```diff
@@ -77,12 +77,11 @@ include_directories(${GLFW_DIR}/include)
 target_link_libraries(${CMAKE_PROJECT_NAME} glfw ${GLFW_LIBRARIES})
 
 # FMOD
-set(FMOD_DIR "$ENV{FMOD_DIR}")
+set(FMOD_DIR "${PROJECT_SOURCE_DIR}/libraries/fmod")
```

Comment:
Can we note treat this as a third party lib or an ext?

Reply:
Are you saying can we "not" treat this? From my perspective our two options are for everyone to either have their own fmod library and set an ENV FMOD_DIR to point to it, or for the libraries to just be included in the project. I don't understand what you mean by not treat it as a 3rd party library since it is a third party library, like glm, glsl, and glfw. I don't have a whole lot of experience working with 3rd party libraries, so I don't know what you expect.

Reply:
I misunderstood what you meant by "can we not treat this." The problem is that fmod doesn't provide the source code, just the header files and libraries. I think having the header files under Thirdparty might work, but we would still have to link with the libraries (.dll, .lib, .so. a)